### PR TITLE
Fix "Save/Delete Layout" text field goes outside of the popup

### LINKED ISSUE #8481: Save Layout text input goes outside of the popup menu
**Operating system or device - Godot version:**
Windows 10 - Godot 3.0

**Issue description:**
Save Layout

### PR DESCRIPTION
**Pull request by melix99**
Fixes #8481.

### CODE REVIEW

**Comment by neikeq** [comment]
I think this should be fixed in [EditorNameDialog](https://github.com/Melix19/godot/blob/master/editor/editor_name_dialog.cpp) by using containers. I don't know, it's been a long time since I fixed one of those dialogs.

**Comment by akien-mga** [comment]
Please use explicit commit logs, this is sadly unreadable in the git log. (Hate GitHub's editor which promotes such bad practices :/)

File: editor/editor_node.cpp [hunk 1]
```diff
@@ -4155,14 +4155,14 @@ void EditorNode::_layout_menu_option(int p_id) {
 			current_option = p_id;
 			layout_dialog->set_title(TTR("Save Layout"));
 			layout_dialog->get_ok()->set_text(TTR("Save"));
-			layout_dialog->popup_centered();
+			layout_dialog->popup_centered(Size2(200, 70));
```
Comment (volzhs) [inline editor/editor_node.cpp]: does it work with different DPI settings?

**Comment by Melix19** [comment]
Ok, I fixed all you said. Sorry for the inconveniences, I'm noob with GitHub and Godot in general. :)

**Comment by akien-mga** [comment]
No problem, it looks good now. I'm just annoyed at GitHub for promoting bad practices, not to you for falling into their trap :P

**Comment by akien-mga** [comment]
I had cherry-picked this issue in 2.1, which introduced a regression (LineEdit would be way too small). Upon investigation, it seems weird to me that the margins are applied to the LineEdit and not to the VBoxContainer; doing it in 2.1 fixed the issue (37eaa9d), but the same code in master reintroduces the original bug.

I wonder if we've not just worked around a more global issue here.

**Pull request merged**